pkg/downloader/manager.go

```diff
@@ -206,6 +206,7 @@ func (m *Manager) downloadAll(deps []*chartutil.Dependency) error {
 	}
 
 	destPath := filepath.Join(m.ChartPath, "charts")
+	tmpPath := filepath.Join(m.ChartPath, "tmpcharts")
```

Comment:
Something also to consider. Would it be better to use [`os.TempDir()`](https://golang.org/pkg/os/#TempDir) instead since we don't actually do anything with this directory?